db/schema.rb

```diff
@@ -14,16 +14,16 @@
 
   create_table "absences", force: :cascade do |t|
     t.integer  "member_id"
-    t.integer  "performanceset_id"
```

Comment:
Ok.... I had really thought I fixed this when I set up staging. Sigh.

Please go ahead and merge this. I have some time tonight and will try to fix the migrations to make this work.

I also added you as a collaborator, let me know if you're not able to merge now. (sorry, should have done yesterday)